src/types/env.ts

```diff
@@ -5,7 +5,7 @@ import { StandardValidator } from "typebox-validators";
 export const envSchema = T.Object({
   SUPABASE_URL: T.String(),
   SUPABASE_KEY: T.String(),
-  UBIQUIBOT_PUBLIC_KEY: T.String(),
+  UBIQUIBOT_PUBLIC_KEY: T.Optional(T.String()), // optional because it's currently disabled
```

Comment:
Ubiquibot term is deprecated 

Maybe just delete

Reply:
we should enable this since it's used for authentication, otherwise anybody can call a plugin

Reply:
Well, now we have two. We have development and production instance so perhaps:

`KERNEL_APP_PUBLIC_KEY` and `DEV_KERNEL_APP_PUBLIC_KEY`

Reply:
Shall be fixed by using the SDK, can come in a later PR I guess.

Reply:
So should I leave all references alone in this PR and another PR will address this public key situation? Multiple comments saying to delete so I'm not sure, I'm going with leaving them alone so it's easier to replace in the other PR. Let me know if I should in fact delete all refs

Reply:
You can delete as it is all handled by the SDK.